File: examples/js/address/lookupPostcode/lookupPostcode.js
const { AddressService } = require("m3o/address");

// Lookup a list of UK addresses by postcode
async function lookupPostcode() {
  let addressService = new AddressService(process.env.MICRO_API_TOKEN);
  let rsp = await addressService.lookupPostcode({
    postcode: "SW1A 2AA",
  });
  console.log(rsp);
}

lookupPostcode();

File: examples/go/address/lookupPostcode/lookupPostcode.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/address"
)

// Lookup a list of UK addresses by postcode
func LookupPostcode() {
	addressService := address.NewAddressService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := addressService.LookupPostcode(&address.LookupPostcodeRequest{
		Postcode: "SW1A 2AA",
	})
	fmt.Println(rsp, err)
}
